doc/cartridge_kubernetes_guide/index.rst

```diff
@@ -0,0 +1,1016 @@
+Running multiple Tarantool Cartridge clusters in different namespaces
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tarantool Kubernetes operator can manage Tarantool Cartridge clusters only in its
+own namespace. Therefore, to deploy multiple Cartridge clusters in
+different namespaces you need to deploy an operator in each of them.
+
+To install an operator in several namespaces, just specify the required
+namespace during installation:
+
+.. code-block:: console
+
+   $ helm install tarantool-operator tarantool/tarantool-operator --namespace NS_1 --create-namespace --version 0.0.8
+
+   $ helm install tarantool-operator tarantool/tarantool-operator --namespace NS_2 --create-namespace --version 0.0.8
+
+These commands set the operator to the namespace ``NS_1`` and the namespace
+``NS_2``. Then, in each of them, you can run a Tarantool Cartridge
+cluster.
+
+.. code-block:: console
+
+   $ helm install -f values.yaml cartridge tarantool/cartridge --namespace NS_1 --version 0.0.8
+
+   $ helm install -f values.yaml cartridge tarantool/cartridge --namespace NS_2 --version 0.0.8
+
+Finally, we have two namespaces. Each has an operator and a Tarantool Cartridge
+cluster.
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Deleting a cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To remove a cluster, execute the following command:
+
+.. code-block:: console
+
+   $ helm uninstall test-app --namespace tarantool
+   ---
+   release "test-app" uninstalled
+
+After a while, all the pods of our application will disappear. Among the
+pods in the ``tarantool`` namespace, only Tarantool Kubernetes operator will
+remain.
+
+.. code-block:: console
+
+   $ kubectl -n tarantool get pods
+   ---
+   NAME                                  READY   STATUS    RESTARTS   AGE
+   tarantool-operator-7dc948c89b-pb9r4   1/1     Running   0          9m45s
+
+If you need to remove Tarantool Kubernetes operator, execute:
+
+.. code-block:: console
+
+   $ helm uninstall tarantool-operator --namespace tarantool
+   ---
+   release "tarantool-operator" uninstalled
+
+.. NOTE::
+
+   ``helm uninstall`` does not remove persistent volumes. To remove
+   them, you need to additionally perform the following:
+
+   .. code-block:: console
+
+      $ kubectl delete pvc --all -n tarantool
+      ---
+      persistentvolumeclaim "www-routers-0-0" deleted
+      persistentvolumeclaim "www-routers-1-0" deleted
+      persistentvolumeclaim "www-storages-0-0" deleted
+
+.. _cartridge_kubernetes_troubleshooting:
+
+--------------------------------------------------------------------------------
+Troubleshooting
+--------------------------------------------------------------------------------
+
+When creating, updating, or scaling a cluster, errors may occur due to
+lack of physical resources.
+
+Let's examine possible error indications, root causes and solutions.
```

Comment:
I made an error in my Lua code and got this:
```
NAME                                  READY   STATUS             RESTARTS   AGE
routers-0-0                           0/1     CrashLoopBackOff   2          3m47s
storages-0-0                          0/1     CrashLoopBackOff   2          3m43s
storages-1-0                          0/1     CrashLoopBackOff   2          3m42s
tarantool-operator-7879d99ccb-gwkb6   1/1     Running            0          36m
```
Is it a common sutiation, should we also mention it?

Reply:
https://sysdig.com/blog/debug-kubernetes-crashloopbackoff/

Reply:
We can add an item to the troubleshooting section. Good observation.